implementations/rust/ockam/ockam_command/src/kafka/producer/create.rs

```diff
@@ -86,7 +86,7 @@ async fn rpc(ctx: Context, (opts, cmd): (CommandGlobalOpts, CreateCommand)) -> c
 
     let msgs = vec![
         format!(
-            "Buildling KafkaConsumer service {}",
```

Comment:
I am wondering if we could refactor the whole consumer/producer commands to just be one module where everything is parametrized by the service kind. Because those too service are actually almost identical. Then any modification to the commands would always stay consistent

Reply:
I was checking this and I don't see why we couldn't do it. I'll create a `good-first` issue with the details!